api/migrations/createWorkspaceSchema.sql

```diff
@@ -240,17 +195,21 @@ BEGIN
         REFERENCES "table" (id) MATCH SIMPLE
         ON UPDATE NO ACTION
         ON DELETE CASCADE,
```

Comment:
```suggestion
        ON DELETE NO ACTION,
```
No ?

Reply:
Well, if a `table` is removed, I prefer remove all its `field`s. 
If not, this will be equivalent as a `ON DELETE RESTRICT`, and so we won't be able to remove a `table`.